app/adapters/publishing_adapter.rb

```diff
@@ -118,6 +120,7 @@ def save_manual_content(manual, republish: false)
       update_type: update_type,
       publishing_app: GdsApiConstants::PublishingApiV2::PUBLISHING_APP,
       rendering_app: GdsApiConstants::PublishingApiV2::RENDERING_APP,
+      change_note: latest_publication ? "#{latest_publication.title} - #{latest_publication.change_note}" : "",
```

Comment:
might we want this as nil or field not set (depending on which the Publishing API plays ball with) when there isn't a latest publication? Not sure if empty string leads to unintentional surprises or not. 

Also wonder if we risk having lots of items with same change note and I can't remember whether that would be a problem or not - I'm guessing specialist publisher already does that though

Reply:
Yep, agreed, updated in 5df0153 to only set the field if the latest publication exists.

I don't think we have that risk as any major publication requires the section to have a change note.